angularjs/angular.d.ts

```diff
@@ -1713,7 +1714,7 @@ declare module angular {
         /**
          * Whether transclusion is enabled. Enabled by default.
          */
-        transclude?: boolean;
+        transclude?: any;
```

Comment:
@mhegazy Here is a link to the docs backing this change: https://docs.angularjs.org/api/ng/directive/ngTransclude#multi-slot-transclusion
Look at the code sample.